02-Context-or-Narrative/State-of-PureScript-Documentation-2019.md

```diff
@@ -17,43 +17,42 @@ It is well-known among the PureScript community that its documentation is lackin
 Outline
 - Explain what "good" documentation is and define a criteria
 - Evaluate PureScript's documentation using that criteria
-- Explain why PureScript's documentation is lacking and what is being done to improve it
+- Explain why PureScript's documentation is seen as lacking
+- Suggest plans to improve the current documentation strategy or process
```

Comment:
I'm being nitpicky here, but I think this should be rendered as "Suggest ways to improve the current documentation situation"

Strategy can be a part of that. Process can be a part of that. However, I don't want to presume we have the 'authority' to make such pronouncements or the 'power' to actually do it.

Reply:
Yeah I think that's appropriately ambiguous at this point. Good idea.